calculator.js

```diff
@@ -1,5 +1,89 @@
-const exampleAdditionInput = {
+const add = (num1, num2) => num1 + num2;
+const subtract = (num1,num2) => num1 - num2;
+const multiply = (num1, num2) => num1 * num2;
+const divide = (num1, num2) => num1 / num2;
+
+const calculate = function(input) {
+  num1 = input.num1;
+  num2 = input.num2;
+  operation = input.operation.toLowerCase();
+
+  if(isNaN(num1) || isNaN(num2)) return "one or both of the numbers are not valid";
+  
+  if(operation === "add" || operation === "+") {
+    return add(num1, num2);
+  } else if(operation === "subtract" || operation === "-" || operation === "minus") {
+    return subtract(num1, num2);
+  } else if(operation === "multiply" || operation === "*") {
+    return multiply(num1, num2);
+  } else if(operation === "divide" || operation === "/") {
+    if(num2 === 0) return "Can't divide by 0!";
+    return divide(num1, num2);
+  } else {
+    return "not a valid operator"
```

Comment:
I like the decision to have this code `return` the calculated value - that would make this program much easier to extend.